template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hdfs/HdfsRoleConfigProviderTest.java

```diff
@@ -169,8 +166,7 @@ void optimizedHDFSDataEncryptHA() {
     }
 
     @Test
-    void optimizedHDFSDataEncryptHAEtitlementDisabled() {
-        when(entitlementService.isSDXOptimizedConfigurationEnabled(anyString())).thenReturn(false);
+    void optimizedHDFSDataEncryptHAYarn() {
```

Comment:
should the test name be changed to reflect the fact YCloud should not get the optimization?